src/python/pants/notes/2.19.x.md

```diff
@@ -1,7 +1,169 @@
 # 2.19.x Release Series
 
+Pants 2 is a fast, scalable, user-friendly build system for codebases of all sizes. It's currently focused on Python, Go, Java, Scala, Kotlin, Shell, and Docker, with support for other languages and frameworks coming soon.
+
+Individuals and companies can now [sponsor Pants financially](https://www.pantsbuild.org/docs/sponsorship).
+
+Pants is an open-source project that is not owned or controlled by any one company or organization, and does incur some expenses. These expenses are managed by Pants Build, a non-profit that was established for this purpose. This non-profit's only source of revenue is sponsorship by individuals and companies that use Pants.
+
+We offer [formal sponsorship tiers for companies](https://www.pantsbuild.org/docs/sponsorship), as well as individual sponsorships via [GitHub](https://github.com/sponsors/pantsbuild).
+
 ## What's New
 
+### Highlights
+
+- Test retries for flaky tests.
+- Using the `parametrize` to set multiple fields at once when generating new groups of targets.
+- `buildx` support in the Docker backend.
+- New backends for running semgrep and openapi-format.
+
+Keep reading to see the details and what's also included.
+
+### Overall
+
+Tests can now be [attempted multiple times](https://www.pantsbuild.org/v2.19/docs/using-pants-in-ci#tip-automatically-retry-failed-tests), if they fail. This can help with limiting the impact of flaky tests. For instance, to require tests to fail three times before the overall `pants test` goal fails, set [`[test].attempts_default = 3`](https://www.pantsbuild.org/v2.19/docs/reference-test#attempts_default). In this case, a very flaky test might fail on the first and second runs but finally pass on the third run, and thus the overall Pants invocation would pass.
```

Comment:
I think this is still only supported by the pytest backend, unless folks have opted other backends in

Reply:
Ah okay, so it should be moved it to the Python section (and make a call out in the plugin section)?

Do we happen to have issues filed for rolling it out to other backends?

Reply:
I think it makes sense to move it to the Python section. We could also keep it overall (since the machinery is there) and add a note that backend support is coming ("Tests can now be retried ... Currently, only pytest supports this and we're adding more backends"). But maybe that's less useful.
I don't think we made issues for adding the other backends, I'll make a tracking issue for it.

Reply:
Thanks for filing https://github.com/pantsbuild/pants/issues/20378. I've adjusted this text in https://github.com/pantsbuild/pants/pull/20310/commits/e0d0508ac52fd6e4e6dd2b4c61aea450de3b74bd, let me know what you think.